src/components/collection/Summary.js

```diff
@@ -110,13 +110,17 @@ function Summary({
             isLoading={isLoading}
           />
           <Typography variant="h6">{t('Category')}</Typography>
-          {categoriesDisplayed?.map((entry) => (
-            <Chip label={t(entry.name)} />
-          ))}
+          {categoriesDisplayed?.size ? (
+            categoriesDisplayed?.map((entry) => <Chip label={t(entry.name)} />)
+          ) : (
+            <Typography variant="body1">{t('None')}</Typography>
+          )}
```

Comment:
Do you prefer showing None or not showing the section at all? I prefer the second option but let me know if you think otherwise.